### ISSUE TITLE
[Beta 0.13.0 RC1] Malformed Database

### ISSUE THREAD

**Issue by LordNex** [blakeblackshear/frigate#9265]
### Describe the problem you are having

I updated to the latest version and I received the following error. I don’t have a problem deleting and rebuilding the database but I thought you guys might want to see. 

'''
frigate  | s6-rc: info: service s6rc-fdholder: starting
frigate  | s6-rc: info: service s6rc-oneshot-runner: starting
frigate  | s6-rc: info: service s6rc-fdholder successfully started
frigate  | s6-rc: info: service s6rc-oneshot-runner successfully started
frigate  | s6-rc: info: service fix-attrs: starting
frigate  | s6-rc: info: service fix-attrs successfully started
frigate  | s6-rc: info: service legacy-cont-init: starting
frigate  | s6-rc: info: service legacy-cont-init successfully started
frigate  | s6-rc: info: service log-prepare: starting
frigate  | s6-rc: info: service log-prepare successfully started
frigate  | s6-rc: info: service nginx-log: starting
frigate  | s6-rc: info: service go2rtc-log: starting
frigate  | s6-rc: info: service frigate-log: starting
frigate  | s6-rc: info: service go2rtc-log successfully started
frigate  | s6-rc: info: service go2rtc: starting
frigate  | s6-rc: info: service nginx-log successfully started
frigate  | s6-rc: info: service frigate-log successfully started
frigate  | s6-rc: info: service go2rtc successfully started
frigate  | s6-rc: info: service go2rtc-healthcheck: starting
frigate  | s6-rc: info: service frigate: starting
frigate  | s6-rc: info: service go2rtc-healthcheck successfully started
frigate  | 2024-01-08 22:03:02.661738795  [INFO] Preparing Frigate...
frigate  | 2024-01-08 22:03:02.671158688  [INFO] Preparing new go2rtc config...
frigate  | s6-rc: info: service frigate successfully started
frigate  | s6-rc: info: service nginx: starting
frigate  | 2024-01-08 22:03:02.703596950  [INFO] Starting NGINX...
frigate  | 2024-01-08 22:03:02.705415780  [INFO] Starting Frigate...
frigate  | s6-rc: info: service nginx successfully started
frigate  | s6-rc: info: service legacy-services: starting
frigate  | s6-rc: info: service legacy-services successfully started
frigate  | 2024-01-08 22:03:03.342500397  [INFO] Starting go2rtc...
frigate  | 2024-01-08 22:03:03.429281817  22:03:03.429 INF go2rtc version 1.8.4 linux/amd64
frigate  | 2024-01-08 22:03:03.430149918  22:03:03.430 INF [api] listen addr=:1984
frigate  | 2024-01-08 22:03:03.430369749  22:03:03.430 INF [rtsp] listen addr=:8554
frigate  | 2024-01-08 22:03:03.430787775  22:03:03.430 INF [webrtc] listen addr=:8555
frigate  | 2024-01-08 22:03:04.492852448  [2024-01-08 22:03:04] frigate.app                    INFO    : Starting Frigate (0.13.0-49814b3)
frigate  | 2024-01-08 22:03:04.608574857  [2024-01-08 22:03:04] peewee_migrate.logs            INFO    : Starting migrations
frigate  | 2024-01-08 22:03:04.708261497  [2024-01-08 22:03:04] peewee_migrate.logs            INFO    : There is nothing to migrate
frigate  | 2024-01-08 22:03:05.109157122  database disk image is malformed
frigate  | 2024-01-08 22:03:06.142685591  [INFO] Service Frigate exited with code 1 (by signal 0)
frigate  | s6-rc: info: service legacy-services: stopping
frigate  | s6-rc: info: service legacy-services successfully stopped
frigate  | s6-rc: info: service nginx: stopping
frigate  | s6-rc: info: service go2rtc-healthcheck: stopping
frigate  | 2024-01-08 22:03:06.159429479  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
frigate  | s6-rc: info: service go2rtc-healthcheck successfully stopped
frigate  | 2024-01-08 22:03:06.228774467  [INFO] Service NGINX exited with code 0 (by signal 0)
frigate  | s6-rc: info: service nginx successfully stopped
frigate  | s6-rc: info: service nginx-log: stopping
frigate  | s6-rc: info: service frigate: stopping
frigate  | s6-rc: info: service frigate successfully stopped
frigate  | s6-rc: info: service go2rtc: stopping
frigate  | s6-rc: info: service frigate-log: stopping
frigate  | s6-rc: info: service nginx-log successfully stopped
frigate  | 2024-01-08 22:03:06.532191079  exit with signal: terminated
frigate  | 2024-01-08 22:03:06.548768207  [INFO] The go2rtc service exited with code 0 (by signal 0)
frigate  | s6-rc: info: service go2rtc successfully stopped
frigate  | s6-rc: info: service go2rtc-log: stopping
frigate  | s6-rc: info: service frigate-log successfully stopped
frigate  | s6-rc: info: service go2rtc-log successfully stopped
frigate  | s6-rc: info: service log-prepare: stopping
frigate  | s6-rc: info: service s6rc-fdholder: stopping
frigate  | s6-rc: info: service log-prepare successfully stopped
frigate  | s6-rc: info: service legacy-cont-init: stopping
frigate  | s6-rc: info: service s6rc-fdholder successfully stopped
frigate  | s6-rc: info: service legacy-cont-init successfully stopped
frigate  | s6-rc: info: service fix-attrs: stopping
frigate  | s6-rc: info: service fix-attrs successfully stopped
frigate  | s6-rc: info: service s6rc-oneshot-runner: stopping
frigate  | s6-rc: info: service s6rc-oneshot-runner successfully stopped
frigate exited with code 0
'''

### Version

0.13.0vrc1

### Frigate config file

```yaml
mqtt:
          # Required: host name
  host: 192.168.1.x
  # Optional: port (default: shown below)
  port: 1883
  # Optional: topic prefix (default: shown below)
  # NOTE: must be unique if you are running multiple instances
  topic_prefix: frigate
  # Optional: client id (default: shown below)
  # NOTE: must be unique if you are running multiple instances
  client_id: frigate
  # Optional: 
  user: user
  # Optional: pas
  # NOTE: MQTT password can be specified with an environment variables that must begin with 'FRIGATE_'.
  #       e.g. password: '{FRIGATE_MQTT_PASSWORD}'
  password: password
  stats_interval: 30
# Optional: Detectors configuration. Defaults to a single CPU detector
detectors:
  # Required: name of the detector
  coral:
    # Required: type of the detector
    # Valid values are 'edgetpu' (requires device property below) and 'cpu'.
    type: edgetpu
    # Optional: device name as defined here: https://coral.ai/docs/edgetpu/multiple-edgetpu/#using-the-tensorflow-lite-python-api
    device: usb
    # Optional: num_threads value passed to the tflite.Interpreter (default: shown below)
    # This value is only used for CPU types
    #num_threads: 3
#  cpu1:
#    type: cpu
#    num_threads: 3
#  cpu2:
#    type: cpu
#    num_threads: 3
# Optional: Database configuration
#database:
  # The path to store the SQLite DB (default: shown below)
 # path: /docker-compose/frigate/config/frigate.db
# Optional: ffmpeg configuration
ffmpeg:
  # Optional: global ffmpeg args (default: shown below)
  global_args: -hide_banner -loglevel warning
  # Optional: global hwaccel args (default: shown below)
  # NOTE: See hardware acceleration docs for your specific device
 # hwaccel_args:
  #  - preset-intel-qsv-h264
  #  - preset-intel-qsv-h265
#    - -c:v
#    - h264_v4l2m2m 
  # Optional: global input args (default: shown below)
  input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1
  output_args:
    # Optional: output args for detect streams (default: shown below)
    detect: -f rawvideo -pix_fmt yuv420p
    # Optional: output args for record streams (default: shown below)
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy -an
    # Optional: output args for rtmp streams (default: shown below)
    rtmp: -c copy -f flv

# Optional: model modifications
model:
  # Optional: path to the model (default: automatic based on detector)
  path: /edgetpu_model.tflite
  # Optional: path to the labelmap (default: shown below)
  labelmap_path: /labelmap.txt
  # Required: Object detection model input width (default: shown below)
  width: 320
  # Required: Object detection model input height (default: shown below)
  height: 320
  # Optional: Label name modifications. These are merged into the standard labelmap.
  labelmap:
    1: face
    2: person
    3: cat
    4: car
    5: license
    6: truck
    7: police
    8: gun
    9: weapon
    10: license_plate
go2rtc:
  streams:
    hallcam:
      - "ffmpeg:http://192.168.1.1/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=user&password=pass#video=copy#audio=copy#audio=opus"
    hallcam_sub1:
      - "ffmpeg:http://192.168.1.1/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=user&password=pass"

cameras:
  hallcam:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/hallcam?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
            - rtmp
            - audio
        - path: rtsp://127.0.0.1:8554/hallcam_sub1?video=copy
          input_args: preset-rtsp-restream
          roles:
            - detect
  livingroom:
    ffmpeg:
      inputs:
        - path: rtsp://user:pass@192.168.20.181/live
          roles:
            - detect
            - rtmp
            - record
            - audio
    detect:
      width: 1920
      height: 1080
    record: # <----- Enable recording
      enabled: True
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true

  sidedoorbell:
    ffmpeg:
#      input_args: -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental -fflags +genpts+discardcorrupt -use_wallclock_as_timestamps 1
      inputs:

        - path: rtsp://user:pass@192.168.10.175:554/cam/realmonitor?channel=1&subtype=1&authbasic=64

          roles:

            - detect

        - path: rtsp://user:pass@192.168.10.175:554/cam/realmonitor?channel=1&subtype=0&authbasic=64

          roles:

            - record
            - audio
            - rtmp

    detect:
      width: 2560
      height: 1920
    record: # <----- Enable recording
      enabled: true
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
    objects:
      filters:
        person:
          mask: 989,918,989,1151


  frontdoor:
    ffmpeg:
      inputs:
        - path: rtsp://user:pass@192.168.10.149/live
          roles:
            - detect
            - rtmp
            - record
            - audio
    detect:
      width: 1920
      height: 1080
    record: # <----- Enable recording
      enabled: True
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
    objects:
      filters:
        person:
          mask:
            - 1354,190,1379,271,1211,255,1191,190
            - 1354,190,1379,271,1455,126,1459,281
  deckcam:
    ffmpeg:
      inputs:
        - path: rtsp://user:pass@192.168.10.69/live
          roles:
            - detect
            - rtmp
            - record
            - audio
    detect:
      width: 1920
      height: 1080
    record: # <----- Enable recording
      enabled: True
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true



# Optional: logger verbosity settings
logger:
  # Optional: Default log verbosity (default: shown below)
  default: info
  # Optional: Component specific logger overrides
  logs:
    frigate.event: debug


# Optional: birdseye configuration
birdseye:
  # Optional: Enable birdseye view (default: shown below)
  enabled: True
  # Optional: Width of the output resolution (default: shown below)
  width: 1920
  # Optional: Height of the output resolution (default: shown below)
  height: 1080
  # Optional: Encoding quality of the mpeg1 feed (default: shown below)
  # 1 is the highest quality, and 31 is the lowest. Lower quality feeds utilize less CPU resources.
  quality: 10
  # Optional: Mode of the view. Available options are: objects, motion, and continuous
  #   objects - cameras are included if they have had a tracked object within the last 30 seconds
  #   motion - cameras are included if motion was detected in the last 30 seconds
  #   continuous - all cameras are included always
  mode: continuous

# Optional: Detect configuration
# NOTE: Can be overridden at the camera level
detect:
  # Optional: width of the frame for the input with the detect role (default: shown below)
  width: 1024
  # Optional: height of the frame for the input with the detect role (default: shown below)
  height: 768
  # Optional: desired fps for your camera for the input with the detect role (default: shown below)
  # NOTE: Recommended value of 5. Ideally, try and reduce your FPS on the camera.
  fps: 7
  # Optional: enables detection for the camera (default: True)
  # This value can be set via MQTT and will be updated in startup based on retained value
  enabled: True
  # Optional: Number of frames without a detection before frigate considers an object to be gone. (default: 5x the frame rate)
  max_disappeared: 25


# Optional: Motion configuration
# NOTE: Can be overridden at the camera level
motion:
  # Optional: The threshold passed to cv2.threshold to determine if a pixel is different enough to be counted as motion. (default: 25)
  # Increasing this value will make motion detection less sensitive and decreasing it will make motion detection more sensitive.
  # The value should be between 1 and 255.
  threshold: 25
  # Optional: Minimum size in pixels in the resized motion image that counts as motion (default: ~0.17% of the motion frame area)
  # Increasing this value will prevent smaller areas of motion from being detected. Decreasing will make motion detection more sensitive to smaller
  # moving objects.
  contour_area: 75
  # Optional: Alpha value passed to cv2.accumulateWeighted when averaging the motion delta across multiple frames (default: 0.2)
  # Higher values mean the current frame impacts the delta a lot, and a single raindrop may register as motion.
  # Too low and a fast moving person wont be detected as motion.
  delta_alpha: 0.2
  # Optional: Alpha value passed to cv2.accumulateWeighted when averaging frames to determine the background (default: 0.2)
  # Higher values mean the current frame impacts the average a lot, and a new object will be averaged into the background faster.
  # Low values will cause things like moving shadows to be detected as motion for longer.
  # https://www.geeksforgeeks.org/background-subtraction-in-an-image-using-concept-of-running-average/
  frame_alpha: 0.2
  # Optional: Height of the resized motion frame  (default: 1/6th of the original frame height, but no less than 180 = 360)
  # This operates as an efficient blur alternative. Higher values will result in more granular motion detection at the expense of higher CPU usage.
  # Lower values result in less CPU, but small changes may not register as motion.
  frame_height: 360
  # Optional: motion mask
  # NOTE: see docs for more detailed info on creating masks


# Optional: Record configuration
# NOTE: Can be overridden at the camera level
record:
  # Optional: Enable recording (default: shown below)
  enabled: true
  # Optional: Number of days to retain recordings regardless of events (default: shown below)
  # NOTE: This should be set to 0 and retention should be defined in events section below
  #       if you only want to retain recordings of events
 # retain_days: 5
  # Optional: Event recording settings
  events:
    # Optional: Maximum length of time to retain video during long events. (default: 600)
    # NOTE: If an object is being tracked for longer than this amount of time, the retained recordings
    #       will be the last x seconds of the event unless retain_days under record is > 0.
#    max_seconds: 1200
    # Optional: Number of seconds before the event to include (default: shown below)
    pre_capture: 10
    # Optional: Number of seconds after the event to include (default: shown below)
    post_capture: 10
    # Optional: Objects to save recordings for. (default: all tracked objects)
    objects:
      - face
      - person
      - cat
      - car
      - truck
      - license
      - police
      - gun
      - license_plate
    # Optional: Restrict recordings to objects that entered any of the listed zones (default: no required zones)
#    required_zones: []
    # Optional: Retention settings for recordings of events
    retain:
      # Required: Default retention days (default: shown below)
      default: 30
      # Optional: Per object retention days
      objects:
        person: 60
        face: 90
        cat: 30
        car: 90
```

### Relevant log output

```shell
frigate  | s6-rc: info: service s6rc-fdholder: starting
frigate  | s6-rc: info: service s6rc-oneshot-runner: starting
frigate  | s6-rc: info: service s6rc-fdholder successfully started
frigate  | s6-rc: info: service s6rc-oneshot-runner successfully started
frigate  | s6-rc: info: service fix-attrs: starting
frigate  | s6-rc: info: service fix-attrs successfully started
frigate  | s6-rc: info: service legacy-cont-init: starting
frigate  | s6-rc: info: service legacy-cont-init successfully started
frigate  | s6-rc: info: service log-prepare: starting
frigate  | s6-rc: info: service log-prepare successfully started
frigate  | s6-rc: info: service nginx-log: starting
frigate  | s6-rc: info: service go2rtc-log: starting
frigate  | s6-rc: info: service frigate-log: starting
frigate  | s6-rc: info: service go2rtc-log successfully started
frigate  | s6-rc: info: service go2rtc: starting
frigate  | s6-rc: info: service frigate-log successfully started
frigate  | s6-rc: info: service nginx-log successfully started
frigate  | s6-rc: info: service go2rtc successfully started
frigate  | s6-rc: info: service go2rtc-healthcheck: starting
frigate  | s6-rc: info: service frigate: starting
frigate  | s6-rc: info: service go2rtc-healthcheck successfully started
frigate  | s6-rc: info: service frigate successfully started
frigate  | s6-rc: info: service nginx: starting
frigate  | 2024-01-08 22:40:06.480489259  [INFO] Preparing Frigate...
frigate  | 2024-01-08 22:40:06.485908179  [INFO] Preparing new go2rtc config...
frigate  | 2024-01-08 22:40:06.492854681  [INFO] Starting NGINX...
frigate  | s6-rc: info: service nginx successfully started
frigate  | s6-rc: info: service legacy-services: starting
frigate  | 2024-01-08 22:40:06.523710928  [INFO] Starting Frigate...
frigate  | s6-rc: info: service legacy-services successfully started
frigate  | 2024-01-08 22:40:07.028852728  [INFO] Starting go2rtc...
frigate  | 2024-01-08 22:40:07.110080810  22:40:07.109 INF go2rtc version 1.8.4 linux/amd64
frigate  | 2024-01-08 22:40:07.110722445  22:40:07.110 INF [api] listen addr=:1984
frigate  | 2024-01-08 22:40:07.110764990  22:40:07.110 INF [rtsp] listen addr=:8554
frigate  | 2024-01-08 22:40:07.111175040  22:40:07.111 INF [webrtc] listen addr=:8555
frigate  | 2024-01-08 22:40:08.117965678  [2024-01-08 22:40:08] frigate.app                    INFO    : Starting Frigate (0.13.0-49814b3)
frigate  | 2024-01-08 22:40:08.233884675  [2024-01-08 22:40:08] peewee_migrate.logs            INFO    : Starting migrations
frigate  | 2024-01-08 22:40:08.436847839  [2024-01-08 22:40:08] peewee_migrate.logs            INFO    : There is nothing to migrate
frigate  | 2024-01-08 22:40:08.812379372  database disk image is malformed
frigate  | 2024-01-08 22:40:09.779618577  [INFO] Service Frigate exited with code 1 (by signal 0)
frigate  | s6-rc: info: service legacy-services: stopping
frigate  | s6-rc: info: service legacy-services successfully stopped
frigate  | s6-rc: info: service nginx: stopping
frigate  | s6-rc: info: service go2rtc-healthcheck: stopping
frigate  | 2024-01-08 22:40:09.797893024  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
frigate  | s6-rc: info: service go2rtc-healthcheck successfully stopped
frigate  | 2024-01-08 22:40:09.853909717  [INFO] Service NGINX exited with code 0 (by signal 0)
frigate  | s6-rc: info: service nginx successfully stopped
frigate  | s6-rc: info: service nginx-log: stopping
frigate  | s6-rc: info: service frigate: stopping
frigate  | s6-rc: info: service frigate successfully stopped
frigate  | s6-rc: info: service go2rtc: stopping
frigate  | s6-rc: info: service frigate-log: stopping
frigate  | 2024-01-08 22:40:09.867550953  exit with signal: terminated
frigate  | 2024-01-08 22:40:09.874694819  [INFO] The go2rtc service exited with code 0 (by signal 0)
frigate  | s6-rc: info: service nginx-log successfully stopped
frigate  | s6-rc: info: service go2rtc successfully stopped
frigate  | s6-rc: info: service go2rtc-log: stopping
frigate  | s6-rc: info: service frigate-log successfully stopped
frigate  | s6-rc: info: service go2rtc-log successfully stopped
frigate  | s6-rc: info: service log-prepare: stopping
frigate  | s6-rc: info: service s6rc-fdholder: stopping
frigate  | s6-rc: info: service log-prepare successfully stopped
frigate  | s6-rc: info: service legacy-cont-init: stopping
frigate  | s6-rc: info: service legacy-cont-init successfully stopped
frigate  | s6-rc: info: service fix-attrs: stopping
frigate  | s6-rc: info: service s6rc-fdholder successfully stopped
frigate  | s6-rc: info: service fix-attrs successfully stopped
frigate  | s6-rc: info: service s6rc-oneshot-runner: stopping
frigate  | s6-rc: info: service s6rc-oneshot-runner successfully stopped
frigate exited with code 0
```

### FFprobe output from your camera

```shell
Na
```

### Frigate stats

```json
Na
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Amcrest AD410, ReoLink RCL-520, Wyze Pan Cam with RSTP firmware, and attempting to get a Tuya Pet Feeder Cam integrated as well

### Any other information that may be helpful

I was working also on trying to get my Tuya Pet Feeder Camera to feed as well. Which may have corrupted the DB I’m not sure. I have it as an entity in HA and it shows up under go2rtc but I’m not sure how to get the local stream from it. I commented out that entire camera but still get the following error. I’m going to try and just delete the frigate.db file and let it rebuild. Worse cast I can always revert to a snapshot of the VM it’s running in and reupgrade.

**Comment by NickM-27** [comment]
what type of storage is the DB saved on?

**Comment by LordNex** [comment]
NAS mapped via SMB through it was at one point on the local server. With the newer systems I've converted everything over to NAS. Actually OpenMedialVault 6 as a VM inside ESXi7.0.3 on my power edge. I have a separate VM of Ubuntu Server LTS running Frigate. Although I've been playing with utilizing ProjectCodeIA's mesh ability but it seem if I install them both on the frigate box, whatever fires up first gets the TPU and the other doesn't fire up. I have one inside my HA install. Was hoping to get it to load and run beside Frigate with the coral TPU in a mesh. But even with mDNS and SSDP enabled for dual stack, I can't get Frigate to see either of the detectors. Which isn't that big of deal as I plan on installing either PCAI or CompreFace locally on a Jetson Nano as soon as I get an SSD to install it on instead of an SD Card.

**Comment by NickM-27** [comment]
it is not recommended to run a DB on network storage for this exact reason

**Comment by LordNex** [comment]
> it is not recommended to run a DB on network storage for this exact reason

Well I didn't move it to there. All I did was follow the changes you guys documented to make in the docker compose file to migrate to the newest setting. I let Frigate decide where to place stuff but it seems like that's what it should be. Give me a bit and I'll put up my docker-compose file and you can look at it

**Comment by LordNex** [comment]
Oh and technically there on the same physical server. We're just talking about different VMs it's only processing the vSwitch traffic which is all CPU and PCI Lane Speed I believe. 

On a side note, I was able to get my Tuya based pet feeder to install on home assistant. And sure enough I can get a stream out of it. I've ran a port scan on it and it doesn't seem to be using any known ports that I'm aware of. My guess is that it's being proxied by Tuya Client, but I'm not sure how to get this stream over to the Frigate VM of Ubuntu Server 22 LTS. It's in one VM, another is Home Assistant. And the 3rd running OpenMediaVault as a NAS. This is running on a 40 core 13th(I think)@ gen Dell PowerEdge R620 with 256 gigs of ram and 4 x 1.2TB SAS drives in RAID 5. The Host OS runs off dual redundant RAID 1 SD Cards and its VMWare ESXi 7.2 U3.

**Comment by NickM-27** [comment]
> Oh and technically there on the same physical server. We're just talking about different VMs it's only processing the vSwitch traffic which is all CPU and PCI Lane Speed I believe.

right, but the issue with hosting the DB on network storage is not the speed, it is how the network storage protocols work.

> I'm not sure how to get this stream over to the Frigate VM of Ubuntu Server 22 LTS

https://github.com/felipecrs/hass-expose-camera-stream-source

**Comment by LordNex** [comment]
> > Oh and technically there on the same physical server. We're just talking about different VMs it's only processing the vSwitch traffic which is all CPU and PCI Lane Speed I believe.
> 
> 
> 
> right, but the issue with hosting the DB on network storage is not the speed, it is how the network storage protocols work.
> 
> 
Yea I have no problem running it locally. Only down side is when I made the VM I made it only 64 gigs I can't expand it easy.Actually I think it got mapped there during a beta swap one time and at the time we were told to run like that. And it worked or I'd have known. But it can. Have its own partition as far as I'm concerned. That's the main reason I built it is to tie all these things together. With as many PCI x 16 lanes as this has I'm tempted to throw in a GPU and M.2 card and have it run the main code with the old NAS as a backup. But time and budget will see. 

OMV does pretty well and it has direct access to the iDRAC and its resources to build the actual cluster. This is why I'm not running the host VM off of those drives. 

I'd have to check but it's either its own proprietary filesystem or EXFAT but has both CIFS and SMB protocols active and both file systems can read the data. It doesn't matter if I'm connecting from a Windows machine, Linux, or my iPhone. And from what I see with the hardware monitor doesn't seem to be that much. 

But we're virtualization on virtualization here so I'm not sure how that gets handled.   __
> 
> > I'm not sure how to get this stream over to the Frigate VM of Ubuntu Server 22 LTS
> 
> 
> 
> https://github.com/felipecrs/hass-expose-camera-stream-source

Thank you very kindly sir. I will be soon putting that to use
